### PR TITLE
[#592] add link rel tag to header to support subscribing to RSS

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
I tested this locally but it didn't have any effect. What did I do wrong?

**Comment by timorthi** [comment]
@andy5995 I tested this branch and it works fine for me with [this extension](https://chrome.google.com/webstore/detail/the-rss-aggregator/ffhafkagcdhnhamiaecajogjcfgienom).

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -29,6 +29,7 @@
     </style>
     <!-- Timepicker UI -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-timepicker/1.10.0/jquery.timepicker.min.js" type="text/javascript"></script>
+    <link rel="alternate" type="application/rss+xml" title="RSS" href="https://medium.com/feed/ifme" />
```
Comment (timorthi) [inline app/views/layouts/application.html.erb]: Perhaps change the title to something more descriptive like "if me blog"? :)

**Comment by andy5995** [comment]
@timorthi thanks for testing. I shouldn't need that extension, I'm using Firefox. On sites where a feed is available, the option "subscribe to this page" becomes available under the bookmarks menu.

I'll make that change to the title attribute.

**Comment by julianguyen** [comment]
@andy5995 What's an example of a site where the RSS feed worked for you without a browser extension?

**Comment by andy5995** [comment]
Any Wordpress blog, here's one:

https://lensgirl53.wordpress.com/

And this site on Medium:
https://medium.com/ifme

[](https://postimg.org/image/b0u5x7i11/)

The screenshot above is what I get when I select "Subscribe to this page".

**Comment by baohouse** [comment]
This was my test; I started up the site locally (the branch in this pull request), accessed Bookmark, and was able to find the **Subscribe To This Page** option.

rss

**Comment by andy5995** [comment]
Ooops, I misunderstood your question from last night. But nm, all is clear now.

Let's not merge this yet.. a little more I wanted to do with it, like add a plain hyperlink to the footer. https://github.com/julianguyen/ifme/issues/592#issuecomment-328239718

I have questions about it though, so I'll check in Slack when I get back to work on this.

**Comment by andy5995** [comment]
I used "rss" as a placeholder here ( get_icon_text() ), for the icon, but I don't know where those icons come from, or how to add a new one for RSS. What should I do here?

`<%= link_to(get_icon_text('rss', t('navigation.rss')), "https://medium.com/feed/ifme", target: "blank") %>`

**Comment by andy5995** [comment]
This is completed, and has passed the checks.

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -15,6 +15,7 @@
 					<div class="table_cell large_padding_right">
 						<span class="footer_heading" id="connect_links"><%= t('shared.footer.connect') %></span>
 						<%= link_to(get_icon_text('envelope', t('common.form.email')), 'mailto:join.ifme@gmail.com') %>
+						<%= link_to(get_icon_text('rss', t('navigation.rss')), "https://medium.com/feed/ifme", target: "blank") %>
```
Comment (julianguyen) [inline app/views/shared/_footer.html.erb]: Mind putting this in alphabet order, so before the Twitter link? :)

File: config/locales/es.yml [hunk 3]
```diff
@@ -31,6 +31,7 @@ es:
    twitter: "Twitter"
    facebook: "Facebook"
    medium: "Medium"
+   rss: 'RSS'
```
Comment (andy5995) [inline config/locales/es.yml]: I wasn't consistent with placement here. I'm going to fix that

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (andy5995) [reply app/views/shared/_footer.html.erb]: oh, sure!

Comment (julianguyen) [reply app/views/shared/_footer.html.erb]: Oh sorry, I meant in this html.erb file!

Comment (andy5995) [reply app/views/shared/_footer.html.erb]: oh, no need to apologize. My bad, I didn't look at the line to which you referred. I'll change that right now.

**Comment by andy5995** [comment]
And now this has patch contains more commits than changed lines. I guess that means I need to slow down a little.

**Pull request merged**